src/chain/kusama/types/crowdloan/calls.ts

```diff
@@ -0,0 +1,23 @@
+import {sts, Block, Bytes, Option, Result, CallType, RuntimeCtx} from '../support'
```

Comment:
Not a big thing but usually there's a space after `{` and before `}` in import statements. As this is not the only occurrence of that it might be a good idea to add some sort of a linter to make sure formatting is taken care of.

Reply:
It's in the types folder so its auto-generated code. In non-auto-generated code, there is prettier.